src/Sulu/Bundle/AdminBundle/Resources/js/components/SingleIconSelect/SingleIcon.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import singleIconSelectStyle from './singleIconSelect.scss';
+
+type Props = {
+    content: string,
+    id: string | number,
+    isSelected: boolean,
+    onClick?: (id: string | number, selected: boolean) => void,
```

Comment:
would remove the `selected` parameter here because this coponent does not know anything about selections